infrastructure/environments/dplplat01/configuration/patch-pr-env-resource-requests/chart/templates/patch-cms-bnf-deployments.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: patch-resource-requests
+  namespace: default
+spec:
+  schedule: "0 */6 * * *" # Every 6 hours
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: patch-resources
+            image: bitnami/kubectl:latest
```

Comment:
Pin to kubernetes version we know works with our setup.

kubectl may update or we may update cluster to create inconsistency unknowingly - so it's better to be able to control it.